sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-cas-send-invoices.sql

```diff
@@ -0,0 +1,20 @@
+-- Running the scheduler for cas send invoice every day Monday to Friday at 1 am, 1 pm and 8 pm UTC / at 5 am, 5 pm and 12 am PST.
+INSERT INTO
+  sims.queue_configurations(queue_name, queue_configuration, queue_settings)
+VALUES
+  (
+    'cas-send-invoices',
+    '{
+        "cron": "0 1,13,20 * * 1-5",
```

Comment:
Was the Wiki updated with the new scheduler? If not, please do it.

Reply:
https://github.com/bcgov/SIMS/wiki/Schedulers

updated